src/double.ts

```diff
@@ -43,7 +43,7 @@ export class Double {
   }
 
   toString(radix?: number): string {
-    return this.value.toString(radix);
+    return radix == null && Object.is(this.value, -0) ? '-0' : this.value.toString(radix);
```

Comment:
why is the radix guard necessary here?  I'd expect that in any radix, negative zero is returned as `-0`.

Reply:
We shouldn't make this change (at least not here) I made it while debugging something sorry it slipped in